sdk/core/core-rest-pipeline/package.json

```diff
@@ -21,8 +21,8 @@
   "types": "core-rest-pipeline.shims.d.ts",
   "scripts": {
     "build:samples": "echo Obsolete",
-    "build:test": "tsc -p . && dev-tool run bundle",
-    "build": "npm run clean && tsc -p . && dev-tool run bundle && api-extractor run --local",
+    "build:test": "npx playwright install",
```

Comment:
Hi! I have some questions 😄 

1. Does this install the playwright browsers? If so, is there an earlier step we can do this work to avoid an expensive operation in every build (that ideally would only run once if I understood it correctly) or will it just no-op quickly if the required browsers are installed?

2. what browsers do we target in our tests? if _only_ chrome and we're ok with that maybe `npx playwright install chrome` can save some setup time? 

Just suggestions, feel free to ignore me 👍

Reply:
> 1. Does this install the playwright browsers? If so, is there an earlier step we can do this work to avoid an expensive operation in every build (that ideally would only run once if I understood it correctly) or will it just no-op quickly if the required browsers are installed?

Yeah, it does the install but it no-ops if they're already installed. I wish there was an option to have it do this as part of running the tests, but I didn't find an option to do so, which is why I hijacked build:test for it.

> 2. what browsers do we target in our tests? if _only_ chrome and we're ok with that maybe `npx playwright install chrome` can save some setup time?

I think only chrome for now, and certainly I'm fine making this smarter, or perhaps we could have some kind of dev-tool magic to centralize this logic? @jeremymeng what do you think?

Reply:
> or perhaps we could have some kind of dev-tool magic to centralize this logic?
yeah similar to what we did to ensure test proxy is downloaded before recoding/playback? we could add a `dev-tool run test:browser` command

Reply:
I wonder whether there's a way to do `npx playwright install chrome` when we install/update dependency, i.e., `rush update`. I feel that's more natural place for installation.

Reply:
pushed an update to use a eventhooks to do this automagically when rush update runs, take a look!

Reply:
I tried only install chrome, but the command would fail if chrome is already installed.  If installing all browsers turns out to be too much work in the future, we could explore dev-tool/script approach to swallow that particular error later.